manifest-generator/generator-services/src/main/java/io/hyscale/generator/services/plugins/VolumeTemplatesHandler.java

```diff
@@ -82,29 +81,32 @@
         serviceMetadata.setAppName(manifestContext.getAppName());
 
         List<ManifestSnippet> snippetList = new ArrayList<>();
-
+        Map<String, String> customLabels = manifestContext
```

Comment:
We can make this is a first-class field in the manifest context?
Gives more visibility to users. What do you say ?

Reply:
I agree, will make those changes.